regression/benchmark_scripts/run_test_and_record.py

```
@@ -0,0 +1,167 @@
+def record_result(
+    test_name, metrics, required_metrics, csv_path, test_type_string
+):
+    
+    file_path = csv_path + "/" + test_name + ".csv"
+
+    bks_metrics = get_bks_change(metrics, required_metrics)
+    # Add default metrics to data
+    required_metrics.update(bks_metrics)
+    metrics.update(bks_metrics)
+
+    req_metrics = list(required_metrics.keys()) + ["date_time", "git_commit"]
```

Ruff (0.14.2)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)